### ISSUE TITLE
10 clay - Lime Attention Detection

### ISSUE THREAD

**Issue by txtr** [CodeBreakMIT/CodeBreak1.0#36]
__10 clay - Lime Attention Detection__

## ℹ️ Project information

- **Project Name**: Lime
- **Short Project Description**: _Attention Detection as a Service for Audio-Visual Content_
- **Team ID**: 10
- **Team Name**: Clay
- **Team Members**: 
1.  Shubham Pednekar https://github.com/txtr
2. Khalid Sheikh https://github.com/shaikhalid
3. Mario D'sa https://github.com/mario8192
4. Amurto Basu https://github.com/amurto

- **Demo Link**: _Upload your project demo on YouTube/Drive and paste its link here_
- **Repository Link**: _https://github.com/zero-structure/clay_
- **Labels**: _Computer Vision, Artificial Intelligence, Data Visualization, Attention Detection_

## 🔥 Your Pitch
Often content creators receive negative reviews after publishing their videos on online video sharing platforms.  Sometimes, their videos remain viewed only for a short duration, after which the viewer is not really interested in the content or may get bored. The income revenue of all these content/video uploaders is highly dependent on the viewing duration of their videos. So, we develop a service where content creators can have their videos evaluated, using the attention detection system we have developed. We collect the video from our client and we have a few trusted individuals watch these videos. We record the faces of those viewers for the duration for which they watch the video. Their facial expressions, gazes and eye pupil movements are detected and analysed by our attention detection model for emotions and we create a dashboard with metrics plotted in charts, where the user can get a grasp of how good the video would perform on a global platform. Thus, he can make changes accordingly before publishing the video. 

## Technology Stack Used
__OpenCV3, Tensorflow - python, Keras, NodeJS, React, Plotly.js, __

## 🔦 Any other specific thing you want to highlight?
_(Optional)_

## ✅ Checklist

**Before you post the issue**:
- [ x] You have followed the issue title format.
- [ x] You have mentioned the correct labels.
- [x ] You have provided all the information correctly.
